peachjam/templates/peachjam/_document_table.html

```diff
@@ -19,8 +19,12 @@
           <td>
             <a href="{{ document.get_absolute_url }}">{{ document.title }}</a>
           </td>
-          {% if doc_table_show_jurisdiction %}
-            <td>{% jurisdiction_icon document %}&nbsp;{{ document.jurisdiction }}</td>
+          {% if MULTIPLE_JURISDICTIONS and MULTIPLE_LOCALITIES %}
```

Comment:
Not quite:

1. if `MULTIPLE_JURISDICTIONS`
  a. show document.country
  b. if document.locality, show it
2. elif `MULTIPLE_LOCALITIES`
  a. if document.locality, show it

Reply:
In the case of MULTIPLE_JURISDICTIONS, do you want the locality showing after the jurisdiction or before? Or should it be a separate column?

Reply:
Same column, but after. Separate with a https://en.wikipedia.org/wiki/Interpunct `·`

eg. `South Africa · Gauteng`

Reply:
Is there a difference between _if document.locality, show it_ and _document.locality|default_if_none:''_? Either way, a document without locality won't display anything

Reply:
Correct, no difference, except that you don't want the middot if there is no locality.